src/sentry/integrations/vercel/integration.py

```diff
@@ -76,6 +77,73 @@ def get_organization_config(self):
 
         return fields
 
+    def update_organization_config(self, mappings):
+        # mappings = {"project_mappings": [[sentry_project_id, vercel_project_id]]}
+
+        metadata = self.model.metadata
+        vercel_client = VercelClient(metadata["access_token"], metadata.get("team_id"))
+        sentry_project_id = mappings["project_mappings"][0][0]
+        vercel_project_id = mappings["project_mappings"][0][1]
+        org = Project.objects.filter(id=sentry_project_id).get()
+        sentry_project = Project.objects.filter(id=sentry_project_id).get()
+        sentry_project_dsn = ProjectKey.objects.filter(project=sentry_project).get()
+
+        data = {
+            "name": "SENTRY_ORG_test",
+            "value": org.slug,
+        }
+        sentry_org_secret = self.create_secret(vercel_client, vercel_project_id, data)
```

Comment:
@ceorourke I think it would be better to use ordered arguments for the name and value instead of passing in data like this. The API client should assemble the payload, we shouldn't do it here.